Tools/vagrant/initvagrant.sh

```diff
@@ -38,6 +38,10 @@ apt-get install -y gdb
 
 sudo -u $VAGRANT_USER ln -fs /vagrant/Tools/vagrant/screenrc /home/$VAGRANT_USER/.screenrc
 
+# enable permissive ptrace:
+perl -pe 's/kernel.yama.ptrace_scope = ./kernel.yama.ptrace_scope = 0/' -i /etc/sysctl.d/10-ptrace.conf
```

Comment:
without sudo I'm surprised either command works.
I also suspect just adding a new line to the end of ptrace.conf would be simpler. I suspect last line wins

Reply:
This script is run as root; the sudo line above these is specifying a non-root user to sudo to to get permissions right and whatnot.